hazelcast/src/main/java/com/hazelcast/instance/BuildInfoProvider.java

```diff
@@ -26,6 +26,7 @@
 
 /**
  * Provides information about current Hazelcast build.
+ * @author Omer Celik
```

Comment:
Nit - many others have worked on this class.

Reply:
I thought it was mandatory to write when we make changes, so I'm removing it immediately. I am sorry